pom.xml

```diff
@@ -3,7 +3,7 @@
 
     <groupId>de.thomaskrille</groupId>
     <artifactId>dropwizard-template-config</artifactId>
-    <version>1.2.1-SNAPSHOT</version>
+    <version>1.3.1-SNAPSHOT</version>
```

Comment:
You can update to `1.3.0-SNAPSHOT`, but `1.3.1-SNAPSHOT` doesn't make any sense :)